develop/CICD/GitHub/GitHubReusableWorkflows/GitHub_Reusable_Workflows_NuGet_Solution_Master_Workflow.md

```diff
@@ -0,0 +1,142 @@
+---
+uid: github_reusable_workflows_nuget_solution_master_workflow
+---
+
+# NuGet solution master workflow
+
+> [!IMPORTANT]
+> This workflow is intended for internal use at Skyline Communications as it uses Skyline-specific validation. However, you can use it as a baseline and adjust it for use elsewhere by replacing the validation step with your own.
+
+The NuGet solution master workflow is designed to run on repositories containing the [DataMiner NuGet Package Solution](xref:Automation_scripts_as_a_Visual_Studio_solution) provided by the DIS extension in Visual Studio or from [Skyline.DataMiner.VisualStudioTemplates](https://www.nuget.org/packages/Skyline.DataMiner.VisualStudioTemplates#readme-body-tab).
```

Comment:
Is this first cross-reference correct? I don't find any mention of the DataMiner NuGet Package Solution on the target page.